src/helpers/gcs.ts

```diff
@@ -27,5 +27,5 @@ export const uploadGCS = (file: Express.Multer.File): string => {
 export const getUrlGCS = (path: string) => {
   const url = `${config.get('gcs.cdn')}/${config.get('node.env')}/${path}`
 
-  return url
+  return path ? url : null
```

Comment:
_ASK_
Why this condition separate when initiating const `url`?

I do prefer this,
```ts
const url = path ? `${config.get('gcs.cdn')}/${config.get('node.env')}/${path}` : null
return url

// or directly

return path ? `${config.get('gcs.cdn')}/${config.get('node.env')}/${path}` : null
```